datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -163,11 +163,11 @@ private boolean shouldSdxBackupBePerformed(SdxCluster cluster) {
         } else if (isVersionOlderThan(cluster, "7.2.1")) {
             retVal = false;
             reason = "Unsupported runtime: " + cluster.getRuntime();
-        } else if (cluster.getCloudStorageFileSystemType().isGcs()) {
+        } else if (cluster.getCloudStorageFileSystemType() == null || cluster.getCloudStorageFileSystemType().isGcs()) {
```

Comment:
please move it into separated branch with proper reason. } else if (cluster.getCloudStorageFileSystemType() == null) { ... }

Reply:
removed the change. I will create new PR for this change.